# Patch for 0.9.14 to adjust configuration
# - Remove -Werror compiler option
#   See https://github.com/neutrinolabs/xrdp/issues/1678
# - Adjust session manager logging configuration
diff -urNad xrdp-0.9.14-orig/configure.ac xrdp-0.9.14/configure.ac
--- xrdp-0.9.14-orig/configure.ac	2020-08-31 09:06:13.000000000 +0300
+++ xrdp-0.9.14/configure.ac	2020-09-04 19:29:51.470823199 +0300
@@ -156,8 +156,8 @@
 AX_CFLAGS_WARN_ALL
 AX_APPEND_COMPILE_FLAGS([-Wwrite-strings])
 
-AM_COND_IF([LINUX],
-  [AX_APPEND_COMPILE_FLAGS([-Werror])]) # bsd has warnings that have not been fixed yet
+#AM_COND_IF([LINUX],
+#  [AX_APPEND_COMPILE_FLAGS([-Werror])]) # bsd has warnings that have not been fixed yet
 
 AM_COND_IF([XRDP_DEBUG],
   [AX_APPEND_COMPILE_FLAGS([-g -O0])],
diff -urNad xrdp-0.9.14-orig/sesman/sesman.ini xrdp-0.9.14/sesman/sesman.ini
--- xrdp-0.9.14-orig/sesman/sesman.ini	2020-08-31 09:33:40.000000000 +0300
+++ xrdp-0.9.14/sesman/sesman.ini	2020-09-04 19:28:08.571543227 +0300
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -65,9 +65,9 @@
 
 [Logging]
 LogFile=xrdp-sesman.log
-LogLevel=DEBUG
+LogLevel=INFO
 EnableSyslog=1
-SyslogLevel=DEBUG
+SyslogLevel=INFO
 
 ;
 ; Session definitions - startup command-line parameters for each session type
diff -urNad xrdp-0.9.14-orig/xrdp/xrdp.ini xrdp-0.9.14/xrdp/xrdp.ini
--- xrdp-0.9.14-orig/xrdp/xrdp.ini	2020-08-31 09:33:40.000000000 +0300
+++ xrdp-0.9.14/xrdp/xrdp.ini	2020-09-04 19:28:08.571543227 +0300
@@ -144,9 +144,9 @@
 
 [Logging]
 LogFile=xrdp.log
-LogLevel=DEBUG
+LogLevel=INFO
 EnableSyslog=true
-SyslogLevel=DEBUG
+SyslogLevel=INFO
 ; LogLevel and SysLogLevel could by any of: core, error, warning, info or debug
 
 [Channels]
